packages/core/src/RenderPipeline/RenderQueue.ts

```diff
@@ -30,14 +48,22 @@ export class RenderQueue {
   }
 
   sortBatch(compareFunc: Function, batcherManager: BatcherManager): void {
-    this._sort(compareFunc);
-    this._batch(batcherManager);
+    this.sort(compareFunc);
+    this.batch(batcherManager);
+  }
+
+  sort(compareFunc: Function): void {
+    Utils._quickSort(this.elements, 0, this.elements.length, compareFunc);
+  }
+
+  batch(batcherManager: BatcherManager): void {
+    batcherManager.batch(this);
   }
 
   render(
     context: RenderContext,
     pipelineStageTagValue: string,
-    maskType: RenderQueueMaskType = RenderQueueMaskType.No
+    stencilOperation: StencilOperation = StencilOperation.Keep
```

Comment:
This parameter is too obscure and not as good as the previous one.

Reply:
OK